.dockerignore

```diff
@@ -2,5 +2,4 @@ vendor/
 docker/mariadb-data
 docker/redis-data
 docker/manticore-data
-storage
-resources
```

Comment:
Removing these from here will lead to docker trying to copy the full storage and resources directories before it can build the image which, when you get gigs of covers and nzbs, can take hours and will likely lead to running out of resources before being able to build.

Reply:
> Removing these from here will lead to docker trying to copy the full storage and resources directories before it can build the image which, when you get gigs of covers and nzbs, can take hours and will likely lead to running out of resources before being able to build.

Sure but we need the directory structure to be inside the base dockerfile especially the resources folder, is there a command that would regenerate it's content ? (i'm not familiar with Laravel), 

Furthermore, I don't see why you would try to build the Dockerfile with content in theses folders.

Reply:
It looks like the big dirs are likely `storage/covers` and `storage/nzb`

I'm not sure if this is valid but maybe add these as:
```
storage/covers/*
stoarge/nzb/*
```